### PR TITLE
Clarify Mars Maths description.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/locales/de/pathfinder_corporations.json [hunk 1]
```diff
@@ -34,9 +34,9 @@
   "Mars Dierct": "Mars Dierkt",
   "(Effect: When you play a Mars tag, you pay 1 M€ less for each Mars tag you have.)": "(EFFEKT: Wenn du ein Marssymbol ausspielst, bezahle 1 M€ weniger je eigenes Marssymbol.",
   "(You start with 52 M€.)": "(Du beginnst mit 52 M€.)",
-  
+
   "Mars Math": "Mars Mathematik",
-  "(Effect: At the beginning of the Research phase, you draw 5 cards to choose from, but do not buy additional cards.)": "(Effekt: Zu Beginn der Forschungsphase ziehst du 5 Karten, aus denen du wählen kannst, aber du kannst keine weiteren Karten kaufen).",
+  "(Effect: At the beginning of the Research phase, you draw 5 cards, but may STILL only buy 4 cards. If you are drafting, keep 2 cards for your first draft.)": "(Effekt: Zu Beginn der Forschungsphase ziehst du 5 Karten, aus denen du wählen kannst, aber du kannst keine weiteren Karten kaufen).",
```
Comment (kberg) [inline src/locales/de/pathfinder_corporations.json:39]: @Borbarad2 would you like to provide updated German text?

Comment (Borbarad2) [reply src/locales/de/pathfinder_corporations.json:39]: Yes, will do it.

Comment (kberg) [reply src/locales/de/pathfinder_corporations.json:39]: OK I was offering to enter it here. But that's fine.

Comment (Borbarad2) [reply src/locales/de/pathfinder_corporations.json:39]: Oh, sorry. I misunderstood the request badly (poor english understanding :) ) sorry. I will provide the updated German text in my next PR myself. Thanks for offering.

**Pull request merged**